packages/framework/presence/src/latestValueManager.ts

```diff
@@ -197,16 +240,37 @@ export interface LatestArguments<T extends object | null> {
 	 * See {@link BroadcastControlSettings}.
 	 */
 	settings?: BroadcastControlSettings | undefined;
+	validator?: StateSchemaValidator<T> | undefined;
 }
 
+/**
+ * Factory for creating a {@link Latest} State object.
+ *
+ * @alpha
+ */
+export function latest<T extends object | null, Key extends string = string>(
+	args: LatestArguments<T> & { validator: StateSchemaValidator<T> },
+): InternalTypes.ManagerFactory<Key, InternalTypes.ValueRequiredState<T>, Latest<T>>;
+
 /**
  * Factory for creating a {@link LatestRaw} State object.
  *
  * @alpha
  */
 export function latest<T extends object | null, Key extends string = string>(
 	args: LatestArguments<T>,
-): InternalTypes.ManagerFactory<Key, InternalTypes.ValueRequiredState<T>, LatestRaw<T>> {
+): InternalTypes.ManagerFactory<Key, InternalTypes.ValueRequiredState<T>, LatestRaw<T>>;
+
+/**
+ * Factory for creating a {@link Latest} or {@link LatestRaw} State object.
+ *
+ * @alpha
+ */
+export function latest<T extends object | null, Key extends string = string>(
+	args: LatestArguments<T>,
```

Comment:
```suggestion
	args: LatestArguments<T> & { validator?: StateSchemaValidator<T> },
```
Alternatively there are two versions of LatestArguments and args here is a union.

Should definitely check for validator in the args and throw not implemented when found unless you are saying it is all working.?

Reply:
Good call on throwing since it's nyi.